velox/functions/prestosql/aggregates/tests/SumTest.cpp

```diff
@@ -462,5 +462,29 @@ TEST_F(SumTest, distinct) {
       .assertResults("SELECT c0, sum(distinct c1) FROM tmp GROUP BY 1");
 }
 
+TEST_F(SumTest, sumWithOrderBy) {
```

Comment:
What is this test supposed to verify? I don't think it verifies that sorting no longer happens. I also assume this test passes with and without the changes in this PR.

Perhaps, change this test to use partialAggregation with ORDER BY and expect the test to succeed because ORDER BY is being ignored.

```
  auto plan =
      PlanBuilder()
          .values({data})
          .partialAggregation(
              {"c0"}, {"sum(c1 ORDER BY c2 DESC)", "avg(c1 ORDER BY c3)"})
          .finalAggregation()
          .planNode();
```

Also, drop ORDER BY from DuckDB query.

Since this test verifies basic functionality of the Aggregation operator, perhaps, move this test to AggregationTest.cpp.

For individual aggregate functions, perhaps, we can have a small unit test that queries the registry and asserts that orderSensitive metadata is set appropriately.

Reply:
Done.